_includes/releases/v22.1/v22.1.0-beta.1.md

```diff
@@ -0,0 +1,121 @@
+- Improved colors for status badges on the Jobs page. Three status on the Jobs page, `cancel-requested`, `pause-requested`, and `revert-failed`, previously had blue status badge colors that were uninformative of their meaning. This commit modifies the badge colors to reflect their meaning. Now `cancel-requested` and `pause-requested` have gray badges and `revert-failed` has a red badge. [#78611][#78611]
+- Fixes a bug where a node in the `UNAVAILABLE` state will not have latency defined and cause the network page to crash. [#78628][#78628]
+
+<h3 id="v22-1-0-beta-1-bug-fixes">Bug fixes</h3>
+
+- CockroachDB might now fetch less rows when performing lookup and index joins on the queries with `LIMIT` clause. [#78473][#78473]
+- Fixed a bug whereby certain catalog interactions which occurred concurrently with node failures were not internally retried. [#78698][#78698]
+- A bug has been fixed which caused the optimizer to generate invalid query plans which could result in incorrect query results. The bug, which has been present since version 21.1.0, can appear if all of the following conditions are true:
+  1. The query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`
+  1. The inner table has an index containing the equality column, like `t2.a` in the example query. 
+  1. The index contains one or more columns that prefix the equality column.
+  1. The prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78972][#78972]
+- The LATEST file that points to the latest full backup in a collection was written to a directory path with the wrong structure. [#78281][#78281]
+
+<h3 id="v22-1-0-beta-1-performance-improvements">Performance improvements</h3>
+
+- Ranges are split and rebalanced during bulk ingestion only when they become full, reducing unnecessary splits and merges. [#78328][#78328]
+- Unused JS files are no longer downloaded when the admin UI loads. [#78665][#78665]
+
+<h3 id="v22-1-0-beta-1-miscellaneous">Miscellaneous</h3>
+
+<h4 id="v22-1-0-beta-1-changes-without-release-note-annotation">Changes without release note annotation</h4>
+
+- release-22.1: sql/rowexec: deflake TestUncertaintyErrorIsReturned (Nathan VanBenschoten <nvanbenschoten@gmail.com>)
+[#78700][#78700] [6832dd1c9][6832dd1c9] 
+
+<h3 id="v22-1-0-beta-1-doc-updates">Doc updates</h3>
+
+{% comment %}Docs team: Please add these manually.{% endcomment %}
+
+<h3 id="v22-1-0-beta-1-contributors">Contributors</h3>
+
+This release includes 82 merged PRs by 40 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
You can delete "We would like to thank..." and all of those users. They're all cockroach employees. I generally just check on slack if I don't already know